<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageRtData</class>
 <widget class="QWidget" name="PageRtData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="tabShape">
        <enum>QTabWidget::Triangular</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Current</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QCustomPlot" name="currentPlot" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Temperature</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,0">
         <item>
          <widget class="QCustomPlot" name="tempPlot" native="true"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QCheckBox" name="tempShowMosfetBox">
             <property name="toolTip">
              <string>Show MOSFET temperature</string>
             </property>
             <property name="text">
              <string>MOSFET</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="tempShowMotorBox">
             <property name="toolTip">
              <string>Show motor temperature</string>
             </property>
             <property name="text">
              <string>Motor</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>RPM</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QCustomPlot" name="rpmPlot" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_4">
        <attribute name="title">
         <string>FOC</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QCustomPlot" name="focPlot" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_5">
        <attribute name="title">
         <string>Rotor Position</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QProgressBar" name="posBar">
             <property name="maximum">
              <number>359</number>
             </property>
             <property name="value">
              <number>120</number>
             </property>
             <property name="format">
              <string>%v Degrees</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCustomPlot" name="posPlot" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QPushButton" name="posInductanceButton">
             <property name="toolTip">
              <string>Start inductance based signal injection detection (BLDC only)</string>
             </property>
             <property name="text">
              <string>Inductance</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="posEncoderButton">
             <property name="toolTip">
              <string>Plot the encoder position</string>
             </property>
             <property name="text">
              <string>Encoder</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="posObserverButton">
             <property name="toolTip">
              <string>Plot the rotor position seen by the observer (FOC only)</string>
             </property>
             <property name="text">
              <string>Observer</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="posPidButton">
             <property name="toolTip">
              <string>Plot the PID controller position</string>
             </property>
             <property name="text">
              <string>PID Pos</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="posEncoderObserverErrorButton">
             <property name="toolTip">
              <string>Plot the difference between the observer and encoder position (FOC only)</string>
             </property>
             <property name="text">
              <string>Obs vs Enc</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="posHallObserverErrorButton">
             <property name="toolTip">
              <string>Plot the difference between the observer and hall sensor position (FOC only)</string>
             </property>
             <property name="text">
              <string>Obs vs Hall</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="posPidErrorButton">
             <property name="toolTip">
              <string>Plot the PID controller position error</string>
             </property>
             <property name="text">
              <string>PID Error</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="posStopButton">
             <property name="toolTip">
              <string>Stop sending position samples</string>
             </property>
             <property name="text">
              <string>Stop</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_7">
        <attribute name="title">
         <string>Experiment</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <item>
          <widget class="ExperimentPlot" name="experimentPlot" native="true"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="buttonFrame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>3</number>
        </property>
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QToolButton" name="autoscaleButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>3</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>Autoscale plots with incoming samples</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../res.qrc">
            <normaloff>:/res/icons/size_off.png</normaloff>
            <normalon>:/res/icons/size_on.png</normalon>:/res/icons/size_off.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="zoomHButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>3</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>Enable horizontal zoom</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../res.qrc">
            <normaloff>:/res/icons/expand_off.png</normaloff>
            <normalon>:/res/icons/expand_on.png</normalon>:/res/icons/expand_off.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="zoomVButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>3</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>Enable vertical zoom</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../res.qrc">
            <normaloff>:/res/icons/expand_v_off.png</normaloff>
            <normalon>:/res/icons/expand_v_on.png</normalon>:/res/icons/expand_v_off.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="rescaleButton">
          <property name="toolTip">
           <string>Rescale plots to fit</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../res.qrc">
            <normaloff>:/res/icons/size_off.png</normaloff>:/res/icons/size_off.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="logRtButton">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Log Realtime Data to File.&lt;/p&gt;&lt;p&gt;Go to &lt;span style=&quot; font-weight:600;&quot;&gt;Settings -&amp;gt; Paths&lt;/span&gt; to choose the output directory.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../res.qrc">
            <normaloff>:/res/icons/rt_off.png</normaloff>
            <normalon>:/res/icons/rt_on.png</normalon>:/res/icons/rt_off.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="RtDataText" name="rtText" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>widgets/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RtDataText</class>
   <extends>QWidget</extends>
   <header>widgets/rtdatatext.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ExperimentPlot</class>
   <extends>QWidget</extends>
   <header>widgets/experimentplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
